functions/time.js

```diff
@@ -0,0 +1,7 @@
+export const noteDate = (time) => {
```

Comment:
You can use a more generic name here, something like `formatDate`.

Reply:
contrariwise I wanted a specific and relative name for this function. but I will do it.